library/src/main/java/com/datatorrent/lib/dimensions/aggregator/Aggregator.java

```diff
@@ -0,0 +1,14 @@
+package com.datatorrent.lib.dimensions.aggregator;
+
+import com.datatorrent.lib.dimensions.AggregateEvent;
+
+import gnu.trove.strategy.HashingStrategy;
+
+public interface Aggregator<EVENT, AGGREGATE extends AggregateEvent> extends HashingStrategy<EVENT>
```

Comment:
There needs to be a consumer of an AggregateEvent, while it is possible to have an AggregateEvent by itself, the interface is meaningless without an Aggregator to operate on it. So I would say that the dependence goes both ways.